pki/certs.go

```diff
@@ -141,16 +147,17 @@ func IssueTestCertificate(
 		return CertificatePair{}, err
 	}
 
-	earliest := clk.Now()
-	latest := earliest.AddDate(0, 0, 90)
+	notBefore := clk.Now()
+	notAfter := notBefore.AddDate(0, 0, 90)
```

Comment:
This will technically generate a cert that is valid for 90 days + 1 second. While that doesn't actually matter in this case, since it's untrusted, it might be good to do the Right Thing here just for the sake of consistency.

Same comment applies on line 159.

Reply:
Yeah, good catch. I have been noodling the idea of making these short lived, or configurable cert lifetimes so we can do things like 1 day certs. I have fixed it to be exactly 90 days "inclusive".